_data/roadmap.yml

```diff
@@ -15,26 +64,7 @@ future:
         description: This includes updating the Kubeflow 1.3.1 openshift stack to work on OCP 4.4+
       - title: Nvidia Triton Server upgrade
         description: This includes upgrading Triton Server that comes with KFServing and creating demos
```

Comment:
I would delete this section since it no longer applies and there is no existing work that needs to be pulled into the future plans

Reply:
@LaVLaS that is not future and is in past.  Do you want it remove?

Reply:
No need to remove it in this PR. I think we'll have to submit a follow-up PR to update that "Past" releases to match the dates/features that were implemented